packages/carbon-component-mapper/src/files/time-pircker-date.d.ts

```diff
@@ -0,0 +1,22 @@
+import { UseFieldApiComponentConfig } from "@data-driven-forms/react-form-renderer";
```

Comment:
Not sure if it is an issue in the original file, but there is a typo in the filename, could you please fix it? I will check v3 later